src/components/CRating/CRating.js

```diff
@@ -25,7 +25,7 @@ const getIconElement = (createElement, index, context) => {
       staticClass: 'px-1',
       key: `icon-${index}`,
       props: {
-        color: self.fillLevel >= index ? 'primary' : '',
+        color: self.fillLevel >= index ? self.config.color : '',
```

Comment:
I think we should always have some default color for filled icons, either in props implementation or in meta.